build_test/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## 0.2.0+1
+## 0.3.0
```

Comment:
Is this actually a breaking change for this package, or is it all internal to the package?